Deploy failures on Travis CI

Now that #11 was merged, some problems are apparent that were nearly impossible to test before (since this Worked On My Machine&trade; and the .travis.yml deploy logic won't run on PRs). Basically there's no `docker-machine` on Travis, so that assumption needs to be fixed:
```console
make: Entering directory `/home/travis/gopath/src/github.com/deis/etcd'
go build -o rootfs/usr/local/bin/boot -a -installsuffix cgo -ldflags "-s -X main.version=v2-alpha" boot.go
make: docker-machine: Command not found
vendor/github.com/prometheus/client_golang/prometheus/registry.go:36:2: cannot find package "bitbucket.org/ww/goautoneg" in any of:
	/home/travis/gopath/src/github.com/deis/etcd/vendor/bitbucket.org/ww/goautoneg (vendor tree)
	/home/travis/.gimme/versions/go1.5.1.linux.amd64/src/bitbucket.org/ww/goautoneg (from $GOROOT)
	/home/travis/gopath/src/bitbucket.org/ww/goautoneg (from $GOPATH)
make: *** [build] Error 1
make: Leaving directory `/home/travis/gopath/src/github.com/deis/etcd'
```

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,6 @@ SHORT_NAME ?= etcd
 export GOARCH ?= amd64
 export GOOS ?= linux
 export MANIFESTS ?= ./manifests
-export DEV_REGISTRY ?= "$(shell docker-machine ip deis 2>&1):5000"
 export DEIS_REGISTRY ?= ${DEV_REGISTRY}/
 
 # Non-optional environment variables

